do not suggest that QA suggestion is always improvement

matkoniecz opened the pull request:
in many cases users thought that all suggestions are improvement

this change attempts to lessen this false impression

matkoniecz committed:
do not suggest that QA suggestion is always improvements

in many cases users thought that all suggestions are improvement

this change attempts to lessen this false impression

matkoniecz commented:
PR triggered by https://community.openstreetmap.org/t/proposed-automated-edit-removal-of-crossing-markings-yes-tags-introduced-in-undiscussed-automated-edits/129614 and https://github.com/openstreetmap/iD/pull/11029

1ec5 commented on data/core.yaml:
```diff
@@ -2066,8 +2066,8 @@ en:
-        title: Upgrade the tags
-        annotation: Upgraded old tags.
+        title: Change the tags
+        annotation: Changed old tags.
```
There’s already another action with a similar title:

https://github.com/openstreetmap/iD/blob/7d6dda002b53efcf3574f49ea9a37eff7800c6d1/data/core.yaml#L86-L87

Instead of “upgrade the tags”, how about “accept the suggested tags”, since that’s what the diff below calls them anyways?

matkoniecz replied:
> There’s already another action with a similar title:

do you know here it is used?

> accept the suggested tags

I would refer to go further away from implying it is a good change, but this is also improvement compared to the current state

1ec5 replied:
`change_tags` is when you manually change a tag. So if you twiddle one of the fields, the Undo button’s tooltip will say “Undo: Changed tags.”

The other potential source of confusion is that we’d wind up with two options for changing the tags. The second option indicates the effect of the change, but the first does not:

<img src="https://github.com/user-attachments/assets/1e7eed7d-d374-48a8-9184-6528482f840d" width="359" alt="Change the tags. Tag as not the same “Volvo”.">

“Complete the tags” would match the warning’s current title, while “Accept the suggested updates” would match the line “Suggested updates” below. But if you find that too optimistic, then we’d need to come up with a replacement for that string too. Personally, I don’t find it too leading, as long as the problem statement in #11029 maintains transparency with the user.

tyrasd replied:
> Accept the suggested updates

Yeah, that sounds like an eloquent solution for this label. :+1:

matkoniecz replied:
changed to this one then

note: I think it is still not making clear that this automated changes may be damaging, but better than what we have now

note: I am not native speaker, I cannot judge is it Engrish or English

SomeoneElseOSM commented:
In my experience iD's suggestions often make things worse (often when a previous tagging was not quite right for a real-life feature, but iD's "suggestions" are far worse).

"changed" is certainly better than "upgraded" - my personal suggestion for wording would likely be much ruder and unlikely to be accepted here :)

matkoniecz commented on data/core.yaml:
```diff
@@ -2066,8 +2066,8 @@ en:
       upgrade_tags:
-        title: Upgrade the tags
-        annotation: Upgraded old tags.
+        title: Change the tags
```
```suggestion
        title: Accept the suggested updates
```

matkoniecz commented on data/core.yaml:
```diff
@@ -2066,8 +2066,8 @@ en:
-        title: Upgrade the tags
-        annotation: Upgraded old tags.
+        title: Change the tags
+        annotation: Changed old tags.
```
```suggestion
        annotation: Accepted the suggested updates
```

matkoniecz committed:
change phrasing

note: I think it is still not making clear that this automated changes may be damaging, but better than what we have now

matkoniecz committed:
Change phrasing

note: I think it is still not making clear that this automated changes may be damaging, but better than what we have now

tyrasd reviewed (approved):
Looks good, but let's merge it after the upcoming release. So that translators have time to localize it.

matkoniecz commented:
> In my experience iD's suggestions often make things worse (often when a previous tagging was not quite right for a real-life feature, but iD's "suggestions" are far worse).

Are they coming from NSI suggestions or from https://github.com/openstreetmap/id-tagging-schema/blob/main/data/deprecated.json or from somewhere else?

(not sure are there issues opened already about these problems)
